raiden_contracts/tests/fixtures/channel_test_values.py

```diff
@@ -1,81 +1,475 @@
 from raiden_contracts.tests.utils import MAX_UINT256, ChannelValues
 
 
+# We must cover the edge cases documented in
+# https://github.com/raiden-network/raiden-contracts/issues/188
+# The scope is to make sure that if someone uses an old balance proof, this cannot be used as
+# an attack to steal tokens.
+# For invalid balance proofs (created and signed with an unofficial Raiden client),
+# we cannot determine and guarantee corectness. There are specific constraints that the
+# Raiden client must enforce that guarantee correctness.
+
+# For all valid last balance proofs provided we will programatically construct a suit of balance
+# proof pairs that could be used for attacks (documented edge cases).
```

Comment:
`For all valid last balance proofs provided we will programatically construct a suit of balance` is outdated, I removed that test.

Reply:
Fixed in https://github.com/raiden-network/raiden-contracts/pull/272